run

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -e
+
+pip install -e .
+
+
+## data scraping and processing
+
+# Multi-country
+python codes/data/multi_country/get_adm_info.py
+Rscript codes/data/multi_country/download_6_countries_JHU.R
+
+# CHN
+python codes/data/china/collate_data.py
+
+# FRA
+Rscript codes/data/france/scrape_conf_cases_by_region.R
+stata -b do codes/data/frgiance/format_infected.do
+stata -b do codes/data/france/format_policy.do
+
+# IRN
+Rscript codes/data/iran/iran_cleaning.R
+python codes/data/iran/iran-split-interim-into-processed.py
+
+# ITA
+python codes/data/italy/italy-download-cases-merge-policies.py
+
+# KOR
+Rscript codes/data/korea/download_and_clean_JHU_southkorea_data.R
```

Comment:
remove - included in download_6 above. Also need to rename in the readme. Consider removing in the readme. Might be worth keeping in case someone want to just look at korea

Reply:
good catch. I think we can remove and people can wade into the code if they really want to just look at korea. DOne